src/widget.tsx

```diff
@@ -81,22 +106,67 @@ function JCPreview(props: PreviewProps): JSX.Element {
   );
 }
 
-/**
- * A React component that renders a single comment or reply.
- *
- * @param comment - the comment object to render. Note: Replies will
- * not be rendered.
- *
- * @param className - a string that will be used as the className of the
- * container element.
- */
+function ReactMarkdownRenderer(props: ReactMarkdownRendererProps): JSX.Element {
+  const {
+    source,
+    latexTypesetter,
+    linkHandler,
+    resolver,
+    sanitizer,
+    shouldTypeset
+  } = props;
+  let node: HTMLElement = document.createElement('div');
+  const [renderElement, SetRenderElement] = React.useState(<div></div>);
+  React.useEffect(() => {
+    const markdownRender = async () => {
+      await renderMarkdown({
+        host: node as HTMLElement,
+        source: source,
+        trusted: false,
+        latexTypesetter: latexTypesetter,
```

Comment:
You don't need to change anything but just so you know when the attribute name is the same as the variable you can just put the attribute name (so instead of `latexTypesetter: latexTypesetter` you can just put `latexTypesetter,`